packages/store/docs/exo-taxonomy.md

```diff
@@ -1,46 +1,55 @@
-Like the big stores, the virtual exo objects are written to external storage outside the JavaScript heap. But these are ephemeral -- they do not survive upgrade. Their only purpose is for high cardinality, so we do not provide a convenience for directly making exo instances. IOW, there is no `makeVirtualExo`, only `defineVirtualExoClass` and `defineVirtualExoClassKit`.
-* ***`*DurableExo*`*** <br>
-The durable exo objects are also written to external storage. These can also survive upgrade, and so can be passed in baggage to a successor vat-incarnation.
+### define*ExoClassKit
+We often call a record of named entangled Xs an "XKit", by analogy to a "toolkit" being a collection of closely related tools. Each call to `defineExoClassKit`, `defineVirtualExoClassKit`, or `defineDurableExoClassKit` defines a kit of entangled Exo classes, and makes and returns a "makeKit" function that makes new instances of that kit. Each instance of the kit is a collection of "facets" that share common encapsulated state.
 
-Note that the total number of exo classes must still low cardinality, even if they are virtual or durable. Being virtual or durable only enables the instances to be high cardinality.
+## Heap vs Virtual vs Durable
 
-This `@endo/exo` package itself exports only the heap variants, and so only exports the names
+### Exo*
+As with stores, the default is that exo objects created by `makeExo` or the functions returned by `defineExoClass` or `defineExoClassKit` live in JavaScript's heap. Therefore, the total number of such Exo objects in a given vat must be able to fit into the JavaScript heap of that vat, and will occupy room in that vat's snapshot. We say the total number of instances is *low cardinality* when we expect the total number to remain low enough that this heap representation is not a problem.
 
-- `makeExo`
-- `defineExoClass`
-- `defineExoClassKit`
+### VirtualExo*
+Like the big stores, the virtual exo objects created by the functions returned by `defineVirtualExoClass` or `defineVirtualExoClassKit` are written to external storage outside the JavaScript heap. But these are ephemeral -- they do not survive upgrade. Their only purpose is for high cardinality, so we do not provide a convenience for directly making exo instances. IOW, there is no `makeVirtualExo`.
+
+### DurableExo*
+The durable exo objects created by the functions returned by `defineDurableExoClass` or `defineDurableExoClassKit` are also written to external storage. These can also survive upgrade, and so can be passed in baggage to a successor vat-incarnation. Note that there is no `makeDurableExo`, although the need is largely covered by `prepareExo`.
 
-The virtual and durable variants are contributed by higher layer packages that build on this one, such as `@agoric/vat-data`.
+### Class Cardinality
+The total number of exo classes must be low cardinality, regardless of virtual/durable status. Being virtual or durable only enables class *instances* to be high cardinality.
```

Comment:
Why must the total number of exo classes be low cardinality?

Reply:
IIRC, because each Kind is associated with a memory footprint.

Reply:
It would be helpful to add an explanation in the doc

Reply:
@warner @erights et al.: before I commit such a change, can you confirm that "memory footprint" is the most important reason to mention for keeping the cardinality of exo classes low?